src/main/java/br/com/sbk/sbking/gui/main/ClientApplicationState.java

```diff
@@ -0,0 +1,37 @@
+package br.com.sbk.sbking.gui.main;
```

Comment:
If you have a better package to place this app state class, please let me know :)

Reply:
Seems ok here.